src/blog/2024/03/http-authentication-node-red-with-flowfuse.md

```diff
@@ -0,0 +1,37 @@
+---
+title: HTTP Node Authentication in Node-RED with FlowFuse
```

Comment:
@knolleary There were some issues with some HTTP endpoints not using the Node-RED authentication, but starting their own server IIRC (Dashboard 2.0, right?) Has that been fully resolved for the whole plugin ecosystem? Or is our authentication still dependant on what nodes / packages are installed?

Reply:
HTTP authentication and authorization this article cover, can we update the title to something more generic?

What about `Securing HTTP traffic for Node-RED with FlowFuse`?

Reply:
```suggestion
title: Securing HTTP Traffic for Node-RED with FlowFuse
```